backend/src/common/services/services.ts

```diff
@@ -1,5 +1,10 @@
+import * as process from 'node:process';
+
+import { JwtService } from '~/common/services/jwt/jwt.service.js';
+
 import { CryptService } from './crypt/crypt.service.js';
 
 const cryptService = new CryptService();
+const jwtService = new JwtService(process.env['JWT_SECRET'] || '');
```

Comment:
I think it's better to add JWT_SECRET constant to `BaseConfig` class in `base-config.package.ts` and then import it from there. This way you can pass `baseConfig` instance to a constructor instead of JWT_SECRET string.

Reply:
In general, idea is good, but I'm not sure that it will be good to import BaseConfig into Services.ts, because for BaseConfig we also need to import a logger.

Reply:
Instead will be better to use declared config I suppose
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/74872088/ba80bbc3-a0e1-423c-800a-8324343a79c6)

Reply:
Import env from here common/config/config.js

Reply:
> Instead will be better to use declared config I suppose ![image](https://private-user-images.githubusercontent.com/74872088/304151290-ba80bbc3-a0e1-423c-800a-8324343a79c6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3NTU1ODQsIm5iZiI6MTcwNzc1NTI4NCwicGF0aCI6Ii83NDg3MjA4OC8zMDQxNTEyOTAtYmE4MGJiYzMtYTBlMS00MjNjLTgwMGEtODMyNDM0M2E3OWM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjEyVDE2MjgwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwODQ5ZjcwMjU5YzQzZTBkMTQyODAzMDNjMTMzMTBmY2Y2YzRmYmVhYWFmNDRhZTUxZGVhYThmM2NiYzJlZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jKbuWVNTsYRoCq9PvZSjHuMl8amc-Ils205VWGjyvJY)

Yeah, this is basically what I've meant. As said above you can use ENV from already defined config instance